tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -615,6 +615,36 @@ describe('util/extension-runners/chromium', async () => {
       }),
   );
 
+  it('does pass port to chrome', async () => {
```

Comment:
This unit test only tests the good-weather behavior. You should also test the bad weather behavior, i.e. the error conditions.

Here are the relevant cases:

- port is available.
- port is clearly invalid, e.g. -1 and 99999
- edge case: port 0. Ordinarily this means to select a random port.
- port is unavailable, which the test can guarantee by reserving a port.